cypher/cypher-docs/src/docs/graphgists/query-tuning/advanced-query-tuning-example.asciidoc

```diff
@@ -290,3 +290,5 @@ Predicates that will not work:
 * Existence (eg. `WHERE exists(n.email)`) because no property type information is given
 
 An existence constraint does not include any type information either and will thus not be enough to trigger _index-backed order by_.
+
+Please note that queries with parameters can *not* trigger _index-backed order by_ in this version of Neo4j.
```

Comment:
This sounds a bit like any parameter in the query will stop us from use Ibob. Maybe it is possible to formulate it so that it is clear that it only applies when the property of the index is a parameter?